CMakeLists.txt

```diff
@@ -11,10 +11,10 @@ include(FetchContent)
 # --------------------------------------------------
 FetchContent_Declare(arcana.cpp
     GIT_REPOSITORY https://github.com/microsoft/arcana.cpp.git
-    GIT_TAG f2757396e80bc4169f2ddb938ce25367a98ffdd0)
+    GIT_TAG f32b40264277391689729c37baa4537d5f6add51)
```

Comment:
This is out of date, we updated arcana.cpp to one commit newer than this last week.